watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -1308,11 +1392,29 @@ def calculate_scaling_factors(self):
                 )
                 iscale.set_scaling_factor(self.flow_mol_phase_comp["Liq", j], sf)
 
+            if self.is_property_constructed("flow_equiv_phase_comp"):
+                if (
+                    iscale.get_scaling_factor(self.flow_equiv_phase_comp["Liq", j])
+                    is None
+                ):
+                    sf = iscale.get_scaling_factor(
+                        self.flow_mol_phase_comp["Liq", j], default=1, warning=True
+                    )
+                    iscale.set_scaling_factor(self.flow_equiv_phase_comp["Liq", j], sf)
+
         # scaling factors for parameters
         for j, v in self.mw_comp.items():
             if iscale.get_scaling_factor(v) is None:
                 iscale.set_scaling_factor(self.mw_comp[j], value(v) ** -1)
 
+        for j, v in self.charge_comp.items():
+            if iscale.get_scaling_factor(v) is None:
+                if value(v) == 0:
+                    sf = 1
+                else:
+                    sf = abs(value(v)) ** -1
+                iscale.set_scaling_factor(self.charge_comp[j], sf)
+
```

Comment:
Is there an apparent reason for this scaling being needed? Did you see errors without this scaling?  In principle there is no need to scale constant parameters, even more so for this charge parameter which always stays a single digit.  I recommend removing this change if no special reason.

Reply:
I didn't see any errors without the scaling. My only thought was since other params like `mw_comp` are scaled that charge should be also. Particularly because the IX model makes heavy use of `conc_equiv`, I would want those variables as well scaled as possible. But your point is taken and will remove in next push.

Reply:
However I will also add that this PR is failing tests right now because of differences in results between the Win and Mac solvers. Could that be a result of poor scaling that removing this change could help?

Reply:
@kurbansitterley If the solver found an optimal solution on both systems but gave different solutions, then the cause is (almost) always poor scaling (more generally, any difference is solver behaviour is likely to be due to scaling).

Reply:
The current version of this PR fails for both Win and Linux but passes MacOS.  Aside from your Ix file, two other failing tests are test_posttreatment and test_lssro_multi_sweep.py.   The draft PR #668  takes over the property package in this PR without adding the scaling on change_comp and is not showing failures on these two files so that suggests removing them is better.   But, that version (in PR #668) still has some issue as I described there.   I recommend that at this PR the IX file be focused only and  investigations on DSPMDE instead be conducted by PR #668.  By this way we may target things a little clearly.   If we agreed,  @kurbansitterley can revoke that charge scaling (where I commented) locally.  That means to have a DSPMDE as in PR #668 .  You can then test your IX file, scaling or other things, locally.  If you find critical thing about DSPMDE, you may mention that under #668.

Reply:
Ok thank you for the feedback and info. I will remove this scaling for `charge_comp`.